Assets/Mirror/Runtime/SyncDictionary.cs

```diff
@@ -0,0 +1,297 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Mirror
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class SyncDictionary<B, T> : IDictionary<B, T>, SyncObject
```

Comment:
what does B, T stand for here? why not K, V for key/value or similar? is this a common C# practice?